testnet/gethnetwork.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM golang:1.17-alpine
+
+# set the base libs to build / run
+RUN apk add build-base bash git linux-headers
+ENV CGO_ENABLED=1
+
+# create the base directory
+RUN mkdir /home/go-obscuro
+
+# cache the go mod packaging
+COPY ./go.mod /home/go-obscuro
+COPY ./go.sum /home/go-obscuro
+WORKDIR /home/go-obscuro
+RUN go get -d -v ./...
+
+# make sure the geth network code is available
+COPY . /home/go-obscuro
+
+# reset any previous geth build
+WORKDIR /home/go-obscuro/integration/gethnetwork/
+RUN rm -rf /home/go-obscuro/integration/.build/geth_bin/ && rm -rf /home/go-obscuro/integration/gethnetwork/geth_bin/
+RUN ./build_geth_binary.sh --version=v1.10.17
```

Comment:
@otherview Can't we reuse the prebuilt binary in the repo? Would be much quicker.

Reply:
No, I don't seem to be able to reuse it. I think it was built with ubuntu maybe ?

Also, there's something funky going on because we have the `geth_bin/geth-v1.10.17` and the `.build../geth-v1.10.17` 
I didn't look much into it, but something is off with the scripts.